src/lib/components/edit-client.svelte

```diff
@@ -180,7 +181,12 @@
 			/>
 			<section class="profile-helper">
 				<Typography variant="small">{$_('profileImageHelper')}</Typography>
-				<Button variant="solid" dimension="small"><Image size={16} />{$_('uploadImage')}</Button>
+				<Button
+					variant="solid"
+					dimension="small"
+					onclick={() => notImplemented(undefined, 'Coming soon!')}
```

Comment:
Please use localised text:
```suggestion
					onclick={() => notImplemented(undefined, $_('Coming soon!'))}
```